brainiak/io.py

```diff
@@ -59,6 +62,10 @@ def load_images_from_dir(in_dir: Union[str, Path], suffix: str = "nii.gz",
     files = sorted(in_dir.glob("*" + suffix))
     for f in files:
         yield nib.load(str(f))
+        logger.info(
+            'file %s is read' %
```

Comment:
`f` is a path here, which is not supported to be the argument of `numpy.load()`, and presumably `nibabel.load()`, for numpy version before 1.12. @mihaic should be able to further justify this.